docs/generate_svgs.js

```diff
@@ -0,0 +1,49 @@
+const fs = require("fs");
+const path = require("path");
+const { draw } = require("memory-viz");
+
+const jsonDirectory = path.resolve(__dirname, "docs/examples");
+const svgDirectory = path.resolve(__dirname, "static/images");
+
+fs.readdirSync(jsonDirectory).forEach((directory) => {
+    const jsonSubDirectory = path.join(jsonDirectory, directory);
+
+    if (fs.statSync(jsonSubDirectory).isDirectory()) {
+        const svgSubDirectory = path.join(svgDirectory, directory);
+        fs.mkdirSync(svgSubDirectory, { recursive: true });
+
+        fs.readdirSync(jsonSubDirectory).forEach((file) => {
+            if (path.extname(file) === ".json") {
+                const jsonPath = path.join(jsonSubDirectory, file);
+                const baseName = path.basename(file, ".json");
+
+                try {
+                    const svgPathLight = path.join(
+                        svgSubDirectory,
+                        `${baseName}_light.svg`
+                    );
+                    if (!fs.existsSync(svgPathLight)) {
```

Comment:
I believe I mentioned this earlier, but the check should not be whether the file exists or not, but rather whether the existing SVG file's timestamp is the same or newer than the corresponding JSON file.